code/game/objects/structures/window.dm

```diff
@@ -15,13 +15,20 @@
 	var/disassembled = 0
 	var/wtype = "glass"
 	var/fulltile = 0
+	var/heat_proof = 0
 	var/list/storeditems = list()
 	var/temp_resistance = 800
 //	var/silicate = 0 // number of units of silicate
 //	var/icon/silicateIcon = null // the silicated icon
 	var/image/crack_overlay
 	can_be_unanchored = 1
 
+/obj/structure/window/BlockSuperconductivity()
```

Comment:
what is this?if this is a new proc you forgot the proc/, and anyway isit even needed?just check if heat_proof isn't 0